Formula/rustscan.rb

```diff
@@ -0,0 +1,23 @@
+class Rustscan < Formula
+  desc "Faster Nmap Scanning with Rust"
+  homepage "https://github.com/rustscan/rustscan"
+  url "https://github.com/RustScan/RustScan/archive/1.3.tar.gz"
+  version "1.3.0"
+  sha256 "3bbaf188fa4014a57596c4d4f928b75bdf42c058220424ae46b94f3a36b61f81"
+  license "MIT"
+  depends_on "rust" => :build
+  depends_on "nmap" => :optional
```

Comment:
We don't support optional dependencies in core tap. Is `nmap` needed at runtime?